packages/runtime/test-runtime-utils/src/mocksForReconnection.ts

```diff
@@ -85,9 +85,9 @@ export class MockContainerRuntimeForReconnection extends MockContainerRuntime {
     private reSubmitMessages() {
         let messageCount = this.pendingMessages.length;
         while (messageCount > 0) {
-            const pendingMessage: IMockContainerRuntimePendingMessage = this.pendingMessages.shift();
+            const pendingMessage: IMockContainerRuntimePendingMessage | undefined = this.pendingMessages.shift();
```

Comment:
Can probably assert that this is not undefined since we already check with messageCount